packages/entities/entities-plugins/src/composables/usePluginMeta.ts

```diff
@@ -1,7 +1,19 @@
 import type { PluginMetaData } from '../types'
 import { PluginGroup, PluginScope } from '../types'
-
 import useI18n from './useI18n'
+import { getColumnFields } from './plugin-schemas/typedefs'
+import { aclSchema } from './plugin-schemas/ACL'
+import aclsCredentialsSchema from './plugin-schemas/credentials/mockedAclSchema.json'
+import { basicAuthSchema } from './plugin-schemas/BasicAuth'
+import basicAuthCredentialsSchema from './plugin-schemas/credentials/mockedBasicAuthSchema.json'
+import { keyAuthSchema } from './plugin-schemas/KeyAuth'
+import keyAuthCredentialsSchema from './plugin-schemas/credentials/mockedKeyAuthSchema.json'
+import { hmacAuthSchema } from './plugin-schemas/HMAC'
+import hmacAuthCredentialsSchema from './plugin-schemas/credentials/mockedHmacAuthSchema.json'
+import { jwtSchema } from './plugin-schemas/JWT'
+import jwtCredentialsSchema from './plugin-schemas/credentials/mockedJwtSchema.json'
+import OAuth2Schema from './plugin-schemas/OAuth2'
```

Comment:
Besides `OAuth2`, KM has another credential type `KeyAuthEnc`:
```suggestion
import OAuth2Schema from './plugin-schemas/OAuth2'
import KeyAuthEncSchema from './plugin-schemas/KeyAuthEnc'
```